js/coinsbit.js

```diff
@@ -0,0 +1,735 @@
+                //             limit: 2,
+                //             total: 334,
+                //             orders: [
+                //                 {
+                //                     id: 8620142523,
+                //                     left: "1.56772",
+                //                     market: "BTC_USDT",
+                //                     amount: "1.56772",
+                //                     type: "limit",
+                //                     price: "46752.63032535",
+                //                     timestamp: 1640031161.544,
+                //                     side: "buy",
+                //                     dealFee: "0",
+                //                     takerFee: "0",
+                //                     makerFee: "0",
+                //                     dealStock: "0",
+                //                     dealMoney: "0"
+                //                 },
+                //                 ...
+                //             ]
+                //         },
+                //         code: 200
+                //     }
+                const resultBids = this.safeValue (responses[0], 'result', {});
+                const resultAsks = this.safeValue (responses[1], 'result', {});
+                const bids = this.safeValue (resultBids, 'orders', []);
+                const asks = this.safeValue (resultAsks, 'orders', []);
+                const response = { 'bids': bids, 'asks': asks };
+                const priceKey = 'price';
+                const amountKey = 'amount';
+                const timestamp = this.milliseconds ();
+                orderbook = this.parseOrderBook (response, symbol, timestamp, 'bids', 'asks', priceKey, amountKey);
+            }
+        }
+        // orderbook['nonce']  <<< TODO >>> -- atm I dont know how to handle that, as the response doesn't have right that property
+        return orderbook;
+    }
+
+    async fetchOHLCV (symbol, timeframe = '1m', since = undefined, limit = undefined, params = {}) {
+        const request = this.handleMarketTypeAndParams ('fetchOHLCV', undefined, params)[1];
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        request['market'] = market['id'];
+        const duration = this.parseTimeframe (timeframe);
+        const maxAllowedDiapason = this.options['maxDiapasonsForTimeframes'][timeframe];
+        let maxRequestedMilliSeconds = null;
+        if (limit === undefined) {
+            maxRequestedMilliSeconds = maxAllowedDiapason * 1000;
+        } else {
+            maxRequestedMilliSeconds = Math.min (this.sum (limit, 1) * duration, maxAllowedDiapason) * 1000;
+        }
+        let response = null;
+        if (market['spot']) {
+            request['interval'] = this.timeframes[timeframe];
+            const now = this.milliseconds ();
+            let startTime = since;
+            let endTime = this.sum (now, 1000); // At first, Let's add one second
```

Comment:
Is there a specific reason for adding one second, and not 0.5 seconds or 1.5 seconds here?

Reply:
Good catch. The reason I thought to add exactly 1 second (and not 0.5 or 1.5) is that in somewhere next lines, we do round down (parseint) the 'now' into seconds amount, which is then passed to 'from/to' parameters.
`request['start'] = parseInt (startTime);`
 So, in theory when user requests 'kline' up till 'now',  I think it had also to include current second.  
 Lmk what you think